var/spack/repos/builtin/packages/gromacs/package.py

```diff
@@ -104,13 +98,28 @@ def cmake_args(self):
         else:
             options.append('-DGMX_GPU:BOOL=OFF')
 
-        simd_value = self.spec.variants['simd'].value
-        if simd_value == 'auto':
-            pass
-        elif simd_value == 'none':
+        simd_features = {
+            'sse2': 'SSE2',
+            'sse4_1': 'SSE4.1',
+            'avx128fma': 'AVX_128_FMA',
+            'avx256': 'AVX_256',
+            'axv128': 'AVX2_128',
+            'avx2_256': 'AVX2_256',
+            'avx512': 'AVX_512',
+            'avx512knl': 'AVX_512_KNL',
+            'qpx': 'IBM_QPX',
+            'ace': 'Sparc64_HPC_ACE',
+            'vmx': 'IBM_VMX',
+            'vsx': 'IBM_VSX',
```

Comment:
We don't currently have support for these flags. Do you know since when they are supported and on which microarchitecture?

Reply:
Drop?

Reply:
I think we can check whether we are on a Power and in that case activate these? So far we go as back as Power 7 and it seems to me that both instruction sets are supported there.

Reply:
Meh, we can add these later back in.